### PR TITLE
API link updates

### PR DESCRIPTION
**Pull request by guardrex**
I noticed these in passing working on some other PRs. A couple of find-and-replace actions should just about lick all of them.

Please send your :eye:👩‍⚕️ bill to me for remittance! :smile:

### CODE REVIEW

File: aspnetcore/data/ef-rp/concurrency.md [hunk 1]
```diff
@@ -73,14 +73,14 @@ Optimistic concurrency includes the following options:
...
+* If the property has been changed after it was fetched, a [DbUpdateConcurrencyException](/dotnet/api/microsoft.entityframeworkcore.dbupdateconcurrencyexception?view=efcore-2.0) is thrown. 
 
 The DB and data model must be configured to support throwing `DbUpdateConcurrencyException`.
 
 ### Detecting concurrency conflicts on a property
 
-Concurrency conflicts can be detected at the property level with the [ConcurrencyCheck](https://docs.microsoft.com/dotnet/api/system.componentmodel.dataannotations.concurrencycheckattribute?view=netcore-2.0) attribute. The attribute can be applied to multiple properties on the model. For more information, see [Data Annotations-ConcurrencyCheck](https://docs.microsoft.com/ef/core/modeling/concurrency#data-annotations).
+Concurrency conflicts can be detected at the property level with the [ConcurrencyCheck](/dotnet/api/system.componentmodel.dataannotations.concurrencycheckattribute?view=netcore-2.0) attribute. The attribute can be applied to multiple properties on the model. For more information, see [Data Annotations-ConcurrencyCheck](https://docs.microsoft.com/ef/core/modeling/concurrency#data-annotations).
```
Comment (scottaddie) [inline aspnetcore/data/ef-rp/concurrency.md]: Remove "https://docs.microsoft.com" from this EF link too.

File: aspnetcore/data/ef-rp/intro.md [hunk 2]
```diff
@@ -311,7 +311,7 @@ In the following code, the `async` keyword, `Task<T>` return value, `await` keyw
 * The `async` keyword tells the compiler to:
 
   * Generate callbacks for parts of the method body.
-  * Automatically create the [Task](https://docs.microsoft.com/dotnet/api/system.threading.tasks.task?view=netframework-4.7) object that's returned. For more information, see [Task Return Type](https://docs.microsoft.com/dotnet/csharp/programming-guide/concepts/async/async-return-types#BKMK_TaskReturnType).
+  * Automatically create the [Task](/dotnet/api/system.threading.tasks.task?view=netframework-4.7) object that's returned. For more information, see [Task Return Type](https://docs.microsoft.com/dotnet/csharp/programming-guide/concepts/async/async-return-types#BKMK_TaskReturnType).
```
Comment (scottaddie) [inline aspnetcore/data/ef-rp/intro.md]: Remove "https://docs.microsoft.com" from this async return types link.

File: aspnetcore/fundamentals/localization.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ By [Rick Anderson](https://twitter.com/RickAndMSFT), [Damien Bowden](https://twi
 
 Creating a multilingual website with ASP.NET Core will allow your site to reach a wider audience. ASP.NET Core provides services and middleware for localizing into different languages and cultures.
 
-Internationalization involves [Globalization](https://docs.microsoft.com/dotnet/api/system.globalization) and [Localization](https://docs.microsoft.com/dotnet/standard/globalization-localization/localization). Globalization is the process of designing apps that support different cultures. Globalization adds support for input, display, and output of a defined set of language scripts that relate to specific geographic areas.
+Internationalization involves [Globalization](/dotnet/api/system.globalization) and [Localization](https://docs.microsoft.com/dotnet/standard/globalization-localization/localization). Globalization is the process of designing apps that support different cultures. Globalization adds support for input, display, and output of a defined set of language scripts that relate to specific geographic areas.
```
Comment (scottaddie) [inline aspnetcore/fundamentals/localization.md]: Remove "https://docs.microsoft.com" from this localization link.

File: aspnetcore/fundamentals/middleware/extensibility-third-party-container/sample/README.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
 # ASP.NET Core Middleware Extensibility Sample
 
-This sample illustrates the use of [IMiddleware](https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.http.imiddleware) and [IMiddlewareFactory](https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.http.imiddlewarefactory) with a 3rd party dependency injection container, [Simple Injector](https://github.com/simpleinjector/SimpleInjector). This sample demonstrates the features described in [Middleware activation with a third-party container in ASP.NET Core](https://docs.microsoft.com/aspnet/core/fundamentals/middleware/extensibility-third-party-container).
+This sample illustrates the use of [IMiddleware](https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.http.imiddleware) and [IMiddlewareFactory](https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.http.imiddlewarefactory) with a 3rd party dependency injection container, [Simple Injector](https://simpleinjector.org). This sample demonstrates the features described in [Middleware activation with a third-party container in ASP.NET Core](https://docs.microsoft.com/aspnet/core/fundamentals/middleware/extensibility-third-party-container).
```
Comment (scottaddie) [inline aspnetcore/fundamentals/middleware/extensibility-third-party-container/sample/README.md]: Remove "https://docs.microsoft.com" from these 3 links.

File: aspnetcore/security/authorization/razor-pages-authorization/sample/README.md [hunk 5]
```diff
@@ -8,10 +8,9 @@ When running the sample, use the email address **maria.rodriguez@contoso.com** t
...
-|        [AuthorizeFolder](https://docs.microsoft.com/dotnet/api/microsoft.extensions.dependencyinjection.pageconventioncollectionextensions.authorizefolder)        |      Adds an [AuthorizeFilter](https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.mvc.authorization.authorizefilter) to all of the pages in a folder with the specified path.      |
-|   [AllowAnonymousToPage](https://docs.microsoft.com/dotnet/api/microsoft.extensions.dependencyinjection.pageconventioncollectionextensions.allowanonymoustopage)   |            Adds an [AllowAnonymousFilter](https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.mvc.authorization.allowanonymousfilter) to a page with the specified path.            |
+| Feature | Description |
+| --- | --- |
+| [AuthorizePage](https://docs.microsoft.com/dotnet/api/microsoft.extensions.dependencyinjection.pageconventioncollectionextensions.authorizepage) | Adds an [AuthorizeFilter](https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.mvc.authorization.authorizefilter) to the page with the specified path. |
```
Comment (scottaddie) [inline aspnetcore/security/authorization/razor-pages-authorization/sample/README.md]: Remove "https://docs.microsoft.com" from the links in this table.

File: aspnetcore/tutorials/razor-pages/search.md [hunk 6]
```diff
@@ -36,7 +36,7 @@ If the `searchString` parameter contains a string, the movies query is modified
...
 
-**Note:** The [Contains](https://docs.microsoft.com//dotnet/api/system.data.objects.dataclasses.entitycollection-1.contains) method is run on the database, not in the C# code. The case sensitivity on the query depends on the database and the collation. On SQL Server, `Contains` maps to [SQL LIKE](https://docs.microsoft.com/sql/t-sql/language-elements/like-transact-sql), which is case insensitive. In SQLite, with the default collation, it's case sensitive.
+**Note:** The [Contains](/dotnet/api/system.data.objects.dataclasses.entitycollection-1.contains) method is run on the database, not in the C# code. The case sensitivity on the query depends on the database and the collation. On SQL Server, `Contains` maps to [SQL LIKE](https://docs.microsoft.com/sql/t-sql/language-elements/like-transact-sql), which is case insensitive. In SQLite, with the default collation, it's case sensitive.
```
Comment (scottaddie) [inline aspnetcore/tutorials/razor-pages/search.md]: Remove "https://docs.microsoft.com" from this transact SQL link.

File: aspnetcore/fundamentals/middleware/extensibility-third-party-container/sample/README.md (hunk 4 above)
Comment (guardrex) [reply aspnetcore/fundamentals/middleware/extensibility-third-party-container/sample/README.md]: To make these README.md file links work with a downloaded sample, I think we should provide the full link.

File: aspnetcore/security/authorization/razor-pages-authorization/sample/README.md (hunk 5 above)
Comment (guardrex) [reply aspnetcore/security/authorization/razor-pages-authorization/sample/README.md]: Ditto

Comment (scottaddie) [reply aspnetcore/security/authorization/razor-pages-authorization/sample/README.md]: Sorry, my eyes were burning at this point. I didn't notice it's a README file. I agree.

**Comment by guardrex** [comment]
ei ei ei ... I found even *MORE* when I updated. Let's just call this a *good first pass* for now. We can make another pass later when there's more free time to goof off. 🏖

**Pull request merged**